apps/api/v2/src/ee/event-types/event-types_2024_04_15/inputs/update-event-type.input.ts

```diff
@@ -177,6 +177,14 @@ export class BookingField_2024_04_15 {
   @ValidateNested({ each: true })
   @Type(() => Source)
   sources?: Source[];
+
+  @IsOptional()
+  @IsNumber()
+  maxLength?: number;
```

Comment:
I think that `maxLength` and `minLength` should not be defined on the input object, but on the level of individual booking field, be it `class BookingField_2024_04_15` or `TextField_2024_06_14` within `packages/platform/types/event-types/event-types_2024_06_14/inputs/booking-fields.input.ts`.

Same logic applies as comment above - we need to also handle the `2024_06_14` release of event-types.